src/frontends/paddle/tests/test_models/gen_scripts/generate_silu.py

```diff
@@ -0,0 +1,40 @@
+#
+# silu paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle as pdpd
+import sys
+
+
+def silu(name: str, x, data_type):
+    pdpd.enable_static()
+
+    with pdpd.static.program_guard(pdpd.static.Program(), pdpd.static.Program()):
+        node_x = pdpd.static.data(
+            name='input_x', shape=x.shape, dtype=data_type)
+        out = pdpd.fluid.layers.silu(x=node_x, name='silu')
+
+        cpu = pdpd.static.cpu_places(1)
+        exe = pdpd.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(pdpd.static.default_startup_program())
+
+        outs = exe.run(
+            feed={'input_x': x},
+            fetch_list=[out])
+
+        saveModel(name, exe, feedkeys=['input_x'], fetchlist=[out],
+                  inputs=[x], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def main():
+    data_type = 'float32'
+    x = np.random.randn(2, 3).astype(data_type)
```

Comment:
Can you test it with more data type and input shape ?

Reply:
I added more test cases with different shapes. Silu only supports float32 and float64 in paddle, but when I try to add float64 test cases, there is the following error message.

```
[ RUN      ] PaddleFuzzyOpTest/FrontEndFuzzyOpTest.testOpFuzzy/paddle_silu_test3
unknown file: Failure
C++ exception with description "not supported dtype inf64" thrown in the test body.
[  FAILED  ] PaddleFuzzyOpTest/FrontEndFuzzyOpTest.testOpFuzzy/paddle_silu_test3, where GetParam() = ("paddle", "test_model_zoo/paddle_test_models/", "silu_test3") (12 ms)
```

And I found that all test cases in paddle op_fuzzy have no float64 type, is there any reason for this?

Reply:
yes our test framework do not support it now